skyportal/enum_types.py

```diff
@@ -1,16 +1,44 @@
-import sqlalchemy as sa
-from enum import Enum
+import astropy.units as u
 import inspect
+import numpy as np
+import sncosmo
+from sncosmo.bandpasses import _BANDPASSES
+from sncosmo.magsystems import _MAGSYSTEMS
+import sqlalchemy as sa
 
+from enum import Enum
 from baselayer.app.env import load_env
-
+from baselayer.log import make_log
 from . import facility_apis
 
-from sncosmo.bandpasses import _BANDPASSES
-from sncosmo.magsystems import _MAGSYSTEMS
+log = make_log('enum_types')
 
 _, cfg = load_env()
 
+# load additional bandpasses into the SN comso registry
+existing_bandpasses_names = [val['name'] for val in _BANDPASSES.get_loaders_metadata()]
+additional_bandpasses_names = []
+for additional_bandpasses in cfg.get('additional_bandpasses', []):
+    name = additional_bandpasses.get("name")
```

Comment:
Curious why you are using `get` instead of dictionary indexing; still correct, obviously.

Reply:
because I want to make sure I get a list if this section is missing in the config.

Reply:
done.

Reply:
Yeah, sorry, my brain short circuited; I forgot that `get` always returns item or None.